rskj-core/src/test/java/co/rsk/peg/BridgeSupportAddSignatureTest.java

```diff
@@ -735,6 +747,106 @@ void addSignature(ActivationConfig.ForBlock activations, FederationMember federa
         );
     }
 
+    @Test
+    void addSignature_correctFederator_whenFedIsSegwit_shouldSign() throws IOException {
```

Comment:
```suggestion
    void addSignature_whenCorrectedFederatorAndFedIsSegwit_shouldSign() throws IOException {
```

The other naming seems weird.

Reply:
what does corrected federator mean?

Reply:
I thought it was a typo. Re-read it again, and now I'm confused about what "correct federator" refers to, what is it?

Reply:
a federator that can sign jaja